public/branding-config-dev.json

```diff
@@ -1,10 +1,10 @@
 {
-  "backgroundColor": "#000000",
+  "X-backgroundColor": "#000000",
```

Comment:
can we get rid of the two changes in this file? I'm not sure how they keep showing up.